ckan/templates-midnight-blue/snippets/language_selector.html

```diff
@@ -1,17 +1,17 @@
 {% set current_lang = request.environ.CKAN_LANG %}
+{% set url_args = request.args.to_dict() %}
+{% do url_args.update(request.view_args) %}
```

Comment:
mixing a MultiDict of request parameters with view arguments seems like a really bad practice.

The MultiDict may have multiple values for the same parameter that will be lost and may conflict with view args names (again, losing parameter values)

What about having a specific parameter for the MultiDict parameters to `url_for` instead to keep these different values separate?